frontend/database/general.py

```diff
@@ -159,9 +160,23 @@ def userExist(email):
 
 
 def getCategoryNames():
-    '''Returns list of article categories available in the database'''
+    """Returns list of article categories available in the database"""
     cur = getDb().cursor()
     cur.execute('SELECT category_ID,category_name FROM categories')
     data = cur.fetchall()
     cur.close()
     return [[x[0], x[1]] for x in data]
+
+
+def insertFeedback(user_id, article_id, type, feedback_text):
+    """Inserts feedback into the database. Returns None if successful and an error if not."""
+    conn = getDb()
+    cur = conn.cursor()
+    sql = 'INSERT INTO feedback (user_ID, article_ID, type, feedback_text) VALUES(%s, %s, %s, %s)'
```

Comment:
Where is feedback_text checked for SQL injections?

Reply:
According to this guide https://realpython.com/prevent-python-sql-injection/   execute should be safe for sql injections as long as parameters are passed as a tuple to the function

Reply:
OK